.travis.yml

```diff
@@ -6,7 +6,7 @@ stages:
   - cov
 
 node_js:
-  - '10'
+  - '12'
```

Comment:
I would recommend we do:

```
node_js:
  - 'lts/*'
  - 'stable'
```

We can do a follow up PR for this!

Reply:
libp2p doesn't allow node12 anymore:
> error libp2p@0.29.0: The engine "node" is incompatible with this module. Expected version ">=12.0.0". Got "10.17.0"